.github/workflows/nightly_conway_dbsync.yaml

```diff
@@ -0,0 +1,19 @@
+name: Nightly tests Conway CC
+
+on:
+  schedule:
+    - cron: '15 05 * * *'
```

Comment:
The workflow needs to have unique startup time. We aim at running max two nightly jobs at the same time, and limit that concurrent runtime as much as possible.